Dockerfile

```diff
@@ -42,11 +42,13 @@ RUN cabal build hap
 RUN upx /hapistrano/dist/build/hap/hap
 
 # Copy Hapistrano to a basic Alpine with SSH
-FROM alpine:3.7
+FROM alpine:3.9
 
 RUN apk update \
  && apk add \
-        openssh-client
+        openssh-client \
+        ca-certificates \
+        git
```

Comment:
@nickovivar Is that why `git` and the other packages wan't installed?

Reply:
Yes, @CristhianMotoche. To bring a better explanation about this, we are using a multistage Dockerfile. In the first part, we build all the hapistrano project with all dependencies and resources, and, in the second stage, we just get the binary of hapistrano. In this step, we were installing only the OpenSSH-client, but we need some other dependencies that will be useful in Circle CI and others similar platforms.